gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ClusterStats.java

```diff
@@ -1,176 +1,173 @@
-  private int runningQueryCount;
-  private int queuedQueryCount;
-  private int blockedQueryCount;
-  private int numWorkerNodes;
-  private boolean healthy;
-  private String clusterId;
-  private String proxyTo;
-  private String externalUrl;
-  private String routingGroup;
-  private Map<String, Integer> userQueuedCount;
-
-  public ClusterStats() {}
-
-  public int getRunningQueryCount()
-  {return this.runningQueryCount;}
-
-  public int getQueuedQueryCount()
-  {return this.queuedQueryCount;}
-
-  public int getBlockedQueryCount()
-  {return this.blockedQueryCount;}
-
-  public int getNumWorkerNodes()
-  {return this.numWorkerNodes;}
-
-  public boolean isHealthy()
-  {return this.healthy;}
-
-  public String getClusterId()
-  {return this.clusterId;}
-
-  public String getProxyTo()
-  {return this.proxyTo;}
-
-  public String getExternalUrl()
-  {return this.externalUrl;}
-
-  public String getRoutingGroup()
-  {return this.routingGroup;}
-
-  public Map<String, Integer> getUserQueuedCount()
-  {return this.userQueuedCount;}
-
-  public void setRunningQueryCount(int runningQueryCount)
-  {this.runningQueryCount = runningQueryCount;}
-
-  public void setQueuedQueryCount(int queuedQueryCount)
-  {this.queuedQueryCount = queuedQueryCount;}
-
-  public void setBlockedQueryCount(int blockedQueryCount)
-  {this.blockedQueryCount = blockedQueryCount;}
-
-  public void setNumWorkerNodes(int numWorkerNodes)
-  {this.numWorkerNodes = numWorkerNodes;}
-
-  public void setHealthy(boolean healthy)
-  {this.healthy = healthy;}
-
-  public void setClusterId(String clusterId)
-  {this.clusterId = clusterId;}
-
-  public void setProxyTo(String proxyTo)
-  {this.proxyTo = proxyTo;}
-
-  public void setExternalUrl(String externalUrl)
-  {this.externalUrl = externalUrl;}
-
-  public void setRoutingGroup(String routingGroup)
-  {this.routingGroup = routingGroup;}
-
-  public void setUserQueuedCount(Map<String, Integer> userQueuedCount)
-  {this.userQueuedCount = userQueuedCount;}
-
-  public boolean equals(final Object o)
-  {
-    if (o == this) {
-      return true;
-    }
-    if (!(o instanceof ClusterStats)) {
-      return false;
-    }
-    final ClusterStats other = (ClusterStats) o;
-    if (!other.canEqual((Object) this)) {
-      return false;
-    }
-    if (this.getRunningQueryCount() != other.getRunningQueryCount()) {
-      return false;
-    }
-    if (this.getQueuedQueryCount() != other.getQueuedQueryCount()) {
-      return false;
+import java.util.Objects;
+
+public class ClusterStats
+{
+    private int runningQueryCount;
+    private int queuedQueryCount;
+    private int blockedQueryCount;
+    private int numWorkerNodes;
+    private boolean healthy;
+    private String clusterId;
+    private String proxyTo;
+    private String externalUrl;
+    private String routingGroup;
+    private Map<String, Integer> userQueuedCount;
+
+    public ClusterStats() {}
+
+    public int getRunningQueryCount()
+    { return this.runningQueryCount; }
```

Comment:
It would be better to format like this:
```java
    {
        return this.runningQueryCount;
    }
```
Same for others.

Reply:
ok ... will do for all of them

Reply:
This comment isn't yet addressed.

Reply:
Dang .. I did the whole code base but not this file somehow.. haha. Pushing now